tasks/src/main/java/ru/sberbank/school/task06/CountMap.java

```diff
@@ -18,21 +18,21 @@
  * <p>

```

Comment:
&lt;p&gt; tag should be preceded with an empty line.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1037664510>*